index.yaml

```diff
@@ -7,8 +7,9 @@ indexes:
 # index.yaml file manually, remove the above marker line (the line
 # saying "# AUTOGENERATED").  If you want to manage some indexes
 # manually, move them above the marker line.  The index.yaml file is
-# automatically uploaded to the admin console when you next deploy
-# your application using appcfg.py.
+# automatically uploaded to Cloud Datastore when you next deploy
```

Comment:
There shouldn't be changes in this file.

Reply:
Fixed.